### ISSUE TITLE
Dosent work on sub cards and bugs

### ISSUE THREAD

**Issue by denperss11** [zsarnett/Custom-Grid-View#14]
Multiply issues with this plugin.. 

1 - The subcard dosent work, it only works on main card.. 
2 - The design gets auto responsive, meaning that when editing at card, for the right design, and then the sceen size ex. changes the cards moves arround and get very mesed.

I would like to get this fixed asap because it's very unstable to currently use.. 

I but for the record, this is a so amazing work u have done, i really apreciate this work! 
- keeep up the good work.. 
When is next update on all other issues ?

**Comment by noxhirsch** [comment]
> I would like to get this fixed asap [...]

That's very rude - this is a project Zack develops in his free time. But I'm sure he can give you a full refund of what you paid for this "addon" if you're not happy.

**Comment by denperss11** [comment]
Hahahahhaha why is it rude?

The question when creating this tricket is..
"
**Describe the solution you'd like**
A clear and concise description of what you want to happen.""

So I did.  But maby u dont know that.

**Comment by noxhirsch** [comment]
> Hahahahhaha why is it rude?
> Did u forget to ready the rest 🤣🤣

No, I didn't. You maybe wrote this with a smile on your face, but that doesn't transport well over text. So just sounds like you demand a fast solution of this problem (which is sadly very common in the open source community - at the cost of the maintainers)

**Comment by zsarnett** [comment]
1. It will never work on sub cards. That doesn't really make any sense. Using Vertical Stacks will only allow manipulating the vertical stack.
2. Yes the view is responsive as you cant keep the same design from desktop to mobile as the cards would be too small.

**Comment by denperss11** [comment]
1 - No what i Mean  is that when u have a menu called ex. Overview then - music, cameras  - lights etc. It only works on Overview menu. Does that make sense ?? 

2 - ex. If I change the cards on a 29 Inch moniter an then look at a 32 inch then the cards are very messed.

**Comment by zsarnett** [comment]
1. Yes it makes sense. You only added it to one view. You need to add the type to all views

2. There are some issues with the responsive behavior. I am exploring options on how to resolve them

**Comment by avviano** [comment]
Hi Zack, thanks for clarifying. I was just going to add a new comment that "Views" in Dashboards (what denperss1 seems to be referring to "sub cards") don't have "Custom-Grid-View" functionality, until I read your comment "need to add type to all views". Thanks.
